guardian/support-frontend
# Add new countdown component for US EOY 2024

## What are you doing in this PR?

Adding a new component to countdown to the US end of year Giving Tuesday event.
This is added to the threeTierLanding page but hidden with the campaign setting isEligible flag.
[**Trello card**](https://trello.com/c/MLXWgpD4/625-add-countdown-to-landing-page)



## Why are you doing this?

This is a feature request for the US End of Year 2024 campaign.

## How to test
Uncomment and update the labelled test CountdownSetting in the usEoy2024 campaign in campaigns.ts to the appropriate date, time for testing the countdown to appear or not.
Run locally and force the flag in the parameter to see the countdown with: 
[https://support.thegulocal.com/us/contribute?forceCampaign=usEoy2024](https://support.thegulocal.com/us/contribute?forceCampaign=usEoy2024).

## How can we measure success?

When the date and time in the settings are correct, can you see the countdown without detriment to the performance of the 3 tier checkout?

## Have we considered potential risks?

Investigated any performance impacts of having a timer updating a component once a second.  Confirmed that only one timer is created.



## Accessibility test checklist


## Screenshots

<img width="1080" alt="ScreenshotOfCountdown" src="https://github.com/user-attachments/assets/33f09ad1-b849-4537-9812-62dcb5f53cab">
